app/uk/gov/hmrc/cataloguefrontend/DateHelper.scala

```diff
@@ -53,6 +53,21 @@ object DateHelper {
       d.format(`dd MMM uuuu HH:mm`)
   }
 
+  implicit class InstantImplicits(d: Instant) {
+    def epochMillis: Long =
+      d.atZone(utc).toInstant.toEpochMilli
+
+    def asPattern(pattern: String): String =
+      d.atZone(utc).format(DateTimeFormatter.ofPattern(pattern))
+
+    def asPattern(dtf: DateTimeFormatter): String =
+      d.atZone(utc).format(dtf)
+
+    def displayFormat: String =
+      d.atZone(utc).format(`dd MMM uuuu HH:mm`)
+  }
+
+
   implicit class InstantToZonedDateTime(instant: Instant) {
```

Comment:
can we move the following into the `InstantImplicits` above?
Wonder if we need all these methods to display Instants in different ways - we should probably just have a a default, and a single `asPattern` function. Can we remove the string overloaded version too, which allows adhoc patterns and is less efficient by creating the DateTimeFormatter each time.